### PR TITLE
exercise_1.12.5

### PR DESCRIPTION
**Pull request by Ifeanyijasper**
This is an example of NameError since the variable "cheese" is not defined.

### CODE REVIEW

File: Ifeanyi/Chapter_1/exercise_1.12.5 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
+[GCC 5.4.0 20160609] on linux
+Type "copyright", "credits" or "license()" for more information.
+>>> cheese
+Traceback (most recent call last):
+  File "<pyshell#0>", line 1, in <module>
+    cheese
+NameError: name 'cheese' is not defined
```
Comment (ch3nkula) [inline Ifeanyi/Chapter_1/exercise_1.12.5]: The focus of this exercise is a "NameError" in python. We know it happens at run-time but there is a name for such an error in python. So update your commit message and PR message.

**Comment by Ifeanyijasper** [comment]
I specified is a NameError at my PR

**Comment by ch3nkula** [comment]
This is what your PR body says;
```
This is to demonstrate an example of run-time error because they do not occur until you run the program.
In this case variable cheese is not defined and also it is specifying that it is a NameError.
```

Do you see you wrote `run-time` error? Go back and read my review comment!

**Comment by Ifeanyijasper** [comment]
OK I have change the PR commit massage.

**Pull request merged**